.ui/library/gviHomeCategoryItem.cpx

```diff
@@ -0,0 +1,81 @@
+{
+	"components": [
+		{
+			"className": ".sf-gridViewItem .gviHomeCategoryItem",
+			"hiddenComponent": true,
+			"id": "c0db-5053-33dd-bd4d",
+			"props": {
+				"children": [
+					"7f2f-e073-0c66-f7b0"
+				],
+				"name": "gviHomeCategoryItem",
+				"parent": "57f4-201f-4bfc-5fc6"
+			},
+			"source": {
+				"page": "__library__"
+			},
+			"type": "GridViewItem",
+			"userProps": {
+				"flexProps": {},
+				"left": "23.40",
+				"top": "30.85"
+			}
+		},
+		{
+			"className": ".sf-flexLayout .gviHomeCategoryItem-flHomeCategoryWrapper",
```

Comment:
this should have been a separate component.

Reply:
done.